conf/lambda.json

```diff
@@ -44,6 +44,9 @@
         "threshold": 0
       }
     },
+    "third_party_libraries": [
```

Comment:
this is confusing, but this instance of `third_party_libraries` is more for if you have custom stuff that is not part of the 'core' of streamalert. since you're adding this as an output to the core, this should actually go here: https://github.com/airbnb/streamalert/blob/b87f7769c1e93f74f7072d92da1961012f928ecb/streamalert_cli/manage_lambda/package.py#L229

and pin the package name/version here:
https://github.com/airbnb/streamalert/blob/b87f7769c1e93f74f7072d92da1961012f928ecb/streamalert_cli/manage_lambda/package.py#L45

Reply:
awesome thanks, i didn't know where to place it, will update now :D